sup3r/pipeline/forward_pass.py

```diff
@@ -693,7 +693,14 @@ def __init__(
         exo_kwargs : dict | None
             Dictionary of args to pass to ExogenousDataHandler for extracting
             exogenous features such as topography for future multistep foward
-            pass
+            pass. This should be a nested dictionary with keys for each
+            exogeneous feature. The dictionaries corresponding to the feature
+            names should include the path to exogenous data source, the
+            resolution of the exogenous data, and how the exogenous data should
+            be used in the model. e.g. {'topography': {'file_paths': 'path to
+            input files', 'source_file': 'path to exo data', 'exo_resolution':
+            {'spatial': '1km', 'temporal': None}, 'steps': [{'model': 0,
+            'combine_type': 'input'}, {'model': 0, 'combine_type': 'layer'}]}
```

Comment:
Related to a previous comment, there's got to be a better way to specify this input. Could we more rigorously define the `ExoExtract` or `ExogenousDataHandler` class to be initialized directly on this input? These kwargs basically get sent to `ExogenousDataHandler` don't they? Let's just put all the parsing into the class and then in this docstring we can just say go look at the args required by the class.